### PR TITLE
SS-7177: Refactor root layout

### PR DESCRIPTION
**Pull request by LemToUp**
@snabela the "absolute" position generates too many unpredictable issues. 
( e.g. open the select list at the https://react-example.shapediver.com/prod/#/modelSelect )

So, I get rid from it and also get rid from aside to control the layout via css Grid system.

### CODE REVIEW

File: src/pages/ExamplePage.module.css [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
-	top: calc(100% - 300px);
-	height: 300px;
-	max-height: 100%;
+.appShellMain {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    height: calc(100vh - 60px);
+    grid-template-areas:
+    "main main main"
```
Comment (LemToUp) [inline src/pages/ExamplePage.module.css:7]: @snabela quite simple grid definition (3x3 area and fill it by predefined lock names)

Comment (snabela) [reply src/pages/ExamplePage.module.css:7]: @LemToUp ah, this is a great solution!

**Review by snabela — CHANGES_REQUESTED** [review]
@LemToUp this is great, many thanks! One problem: On mobile we again get the problem of moving. If I rotate the scene, both the viewport and parameterpanel move sideways as well. Please check.

**Comment by snabela** [comment]
@LemToUp I fixed the problem of the viewport moving when panning: https://github.com/shapediver/ShapeDiverReactExample/pull/40/commits/46aa5a176de5df9be28148629a70abb7c25ba40d

Please check this out: https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action
_Applications using [Touch events](https://developer.mozilla.org/en-US/docs/Web/API/Touch_events) disable the browser handling of gestures by calling [preventDefault()](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault), but should also use touch-action to ensure the browser knows the intent of the application before any event listeners have been invoked._

@MajorMeerkatThe3rd please read this as well. We might implement directly in the viewer to set `touch-action: none` on the viewport container.

**Review by snabela — DISMISSED** [review]

**Comment by snabela** [comment]
> @LemToUp I fixed the problem of the viewport moving when panning: [46aa5a1](https://github.com/shapediver/ShapeDiverReactExample/commit/46aa5a176de5df9be28148629a70abb7c25ba40d)
> 
> Please check this out: https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action _Applications using [Touch events](https://developer.mozilla.org/en-US/docs/Web/API/Touch_events) disable the browser handling of gestures by calling [preventDefault()](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault), but should also use touch-action to ensure the browser knows the intent of the application before any event listeners have been invoked._
> 
> @MajorMeerkatThe3rd please read this as well. We might implement directly in the viewer to set `touch-action: none` on the viewport container.

@LemToUp @MajorMeerkatThe3rd you can test it on your mobiles here: https://react-example.shapediver.com/dev/

**Comment by MajorMeerkatThe3rd** [comment]
> > @LemToUp I fixed the problem of the viewport moving when panning: [46aa5a1](https://github.com/shapediver/ShapeDiverReactExample/commit/46aa5a176de5df9be28148629a70abb7c25ba40d)
> > Please check this out: https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action _Applications using [Touch events](https://developer.mozilla.org/en-US/docs/Web/API/Touch_events) disable the browser handling of gestures by calling [preventDefault()](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault), but should also use touch-action to ensure the browser knows the intent of the application before any event listeners have been invoked._
> > @MajorMeerkatThe3rd please read this as well. We might implement directly in the viewer to set `touch-action: none` on the viewport container.
> 
> @LemToUp @MajorMeerkatThe3rd you can test it on your mobiles here: https://react-example.shapediver.com/dev/

@snabela I'll create a task on the viewer side to investigate this.

**Review by snabela — CHANGES_REQUESTED** [review]
@LemToUp there is a final fix I would ask you to do: The "Accept" and "Reject" buttons are not scrolling together with the parameters, but they should stay fixed on top, such that they can always be clicked.

**Comment by LemToUp** [comment]
> @LemToUp there is a final fix I would ask you to do: The "Accept" and "Reject" buttons are not scrolling together with the parameters, but they should stay fixed on top, such that they can always be clicked.

@snabela done.

**Review by snabela — CHANGES_REQUESTED** [review]
@LemToUp there is a problem with making it sticky, please see this: 
image
We need to make it a separate element which is not part of the scrolling I think. This is how it used to be before the refactoring.

**Comment by LemToUp** [comment]
> @LemToUp there is a problem with making it sticky, please see this: image We need to make it a separate element which is not part of the scrolling I think. This is how it used to be before the refactoring.

Get it. Should be fixed.

**Review by snabela — CHANGES_REQUESTED** [review]
@LemToUp many thanks! There is one problem left: We need to make this change in ExamplePage instead of ViewPage, such that it applies to all our pages.

**Review by snabela — APPROVED** [review]
Many thanks @LemToUp !

File: src/pages/MultipleViewportPage.module.css [hunk 2]
```diff
@@ -9,8 +9,9 @@
...
 }
 
-.asideTabsPanel {
-	flex-grow: 1;
-	overflow: auto;
+.tabs {
+	display: flex;
+	flex-direction: column;
 	max-height: 100%;
 }
```
Comment (snabela) [inline src/pages/MultipleViewportPage.module.css:16]: @LemToUp I think this class is not used

**Pull request merged**